example/pom.xml

```diff
@@ -40,6 +40,11 @@
             <groupId>ch.qos.logback</groupId>
             <artifactId>logback-classic</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.jaegertracing</groupId>
+            <artifactId>jaeger-client</artifactId>
+            <version>0.32.0</version>
+        </dependency>
         <dependency>
```

Comment:
It would be better to write some unit test before writing use case examples. In addition, would you like to analyze your imported dependency and its children dependencies license are compatible with ASL 2.0？

Reply:
ok,I will add some unit tests ,and the dependency of jaeger-client is compatible with ASL 2.0.